pkg/storage/utils/decomposedfs/upload_async_test.go

```diff
@@ -532,5 +532,472 @@ var _ = Describe("Async file uploads", Ordered, func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(parentInfo.Size).To(Equal(uint64(len(file2Content))))
 		})
+
+		It("the first can succeed before the second succeeds", func() {
+			// finish first postprocessing
+			con <- events.PostprocessingFinished{
+				UploadID: uploadID,
+				Outcome:  events.PPOutcomeContinue,
+			}
+			// wait for upload to be ready
+			ev, ok := (<-pub).(events.UploadReady)
+			Expect(ok).To(BeTrue())
+			Expect(ev.Failed).To(BeFalse())
```

Comment:
The tests itself look good. But maybe we could DRY the tests up a bit using some helper methods, so this could turn into something like 
```suggestion
			// finish first postprocessing
			succeedPostprocessing(uploadID);
			
			// or
			failPostprocessing(uploadID);
```
?